Update the base version of develop_customer SDK

fengtai-telink opened the pull request:
#### Summary

- Update BLE libs to BLE release/V4.0.4.4 and compatible with TL721x A3 .
- Change the init clock-frequency of TL321x into 48MHz .
- Fix the PM cannot wake up normally due to incorrect use of macro .
- Fix the damage of data related to deep sleep functionality .

haiwentelink committed:
riscv: telink: add NVS_RAM_OPTIMIZATION to save ram.

- change the nvs ram base to save ram.

Signed-off-by: Haiwen Xia <haiwen.xia@telink-semi.com>

liang4804 committed:
telink: zephyr: move the net buffer from iram to dram

- move the net buffer from iram to dram

Signed-off-by: Liang Zhong <liang.zhong@telink-semi.com>

haiwentelink committed:
riscv: telink: add patches file.

- add patch for csma backoff.

Signed-off-by: Haiwen Xia <haiwen.xia@telink-semi.com>

haiwentelink committed:
telink: B92&TL321X: add exit latency mechanism.

- add suspend & deepretn exit latency mechanism to avoid BLE disconn issue.

Signed-off-by: Haiwen Xia <haiwen.xia@telink-semi.com>

liang4804 committed:
drivers: b9x: update the ieee802154 and soc drivers for tlsr9528a (#450)

-reduce runtime

Signed-off-by: Liang Zhong <liang.zhong@telink-semi.com>

charlie-ai committed:
west.yml: Update Telink HAL hash

- Configure a faster spi clock for flash

Signed-off-by: Lingzhi Chen <lingzhi.chen@telink-semi.com>

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/0c2d784cb4d49a04878a22f5dda589605dd0d4c4 (develop) | https://github.com/telink-semi/hal_telink/commit/3ea94cf921af6ed2fd55b17985904d99dedbbbbf (develop_customer_new_base) | [telink-semi/hal_telink@0c2d784c..3ea94cf9](https://github.com/telink-semi/hal_telink/compare/0c2d784cb4d49a04878a22f5dda589605dd0d4c4..3ea94cf921af6ed2fd55b17985904d99dedbbbbf) |
| mcuboot | https://github.com/telink-semi/mcuboot/commit/1558e7ab0aadb4eac11f03befb5ccd3fa3f0aafe | https://github.com/telink-semi/mcuboot/commit/9c8c7cdfe264e9d07e47c624c91581407bc0c538 (develop_customer) | [telink-semi/mcuboot@1558e7ab..9c8c7cdf](https://github.com/telink-semi/mcuboot/compare/1558e7ab0aadb4eac11f03befb5ccd3fa3f0aafe..9c8c7cdfe264e9d07e47c624c91581407bc0c538) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

fengtai-telink committed:
riscv: telink: add the dual mode proc (#562)

 - Enable gpioa interface .
 - Add the user para partition .
 - Clean the rf radio and dma register before sys_init .
 - Add CONFIG_DUAL_MODE to protect code .
 - Adjust the boards dtsi for jagura, buteo and tercel .
 - Add dual-mode config .
 - Adjust the code .
 - Add customized wdt control .
 - Update west.yml .

Signed-off-by: Fengtai Xie <fengtai.xie@telink-semi.com>

fengtai-telink committed:
riscv: telink: adjust the way to enable automatic watchdog

 - Adjust to CONFIG_WATCHDOG_AUTO .
 - Update the hal hash .

Signed-off-by: Fengtai Xie <fengtai.xie@telink-semi.com>
